src/commands/moderation/prefixCommands/functions/modifyCommand.ts

```diff
@@ -0,0 +1,205 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommand, Logger, makeEmbed, PrefixCommandCategory, refreshSinglePrefixCommandCache, PrefixCommandVersion } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Modify Command - No Connection',
+    description: 'Could not connect to the database. Unable to modify the prefix command.',
+    color: Colors.Red,
+});
+
+const failedEmbed = (commandId: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Command - Failed',
+    description: `Failed to modify the prefix command with id ${commandId}.`,
+    color: Colors.Red,
+});
+
+const wrongFormatEmbed = (invalidString: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Command - Wrong format',
+    description: `The name and aliases of a command can only contain alphanumerical characters, underscores and dashes. "${invalidString}" is invalid.`,
+    color: Colors.Red,
+});
+
+const categoryNotFoundEmbed = (category: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Command - Category not found',
+    description: `The prefix command category ${category} does not exist. Please create it first.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (command: string) => makeEmbed({
+    title: 'Prefix Commands - Modify Command - Does not exist',
+    description: `The prefix command ${command} does not exists. Can not modify it.`,
```

Comment:
```suggestion
    description: `The prefix command ${command} does not exists. Cannot modify it.`,
```
"Cannot" is generally more common. I suggest we use that but obviously that's personal preference and I'm happy to leave it as is!